core/templates/dev/head/pages/header_js_libs.html

```diff
@@ -15,7 +15,8 @@
         automatic: true,
         width: '500px'
       },
-      showMathMenu: false
+      showMathMenu: false,
+      scale: 90
```

Comment:
The text looks just a bit smaller than outside text, try 92 or 93 and see how it goes. Also, in the interest of organization, it would be nice if you could place `scale` before `showMathMenu`.

![image](https://cloud.githubusercontent.com/assets/5758256/21371303/61eeaae0-c6c5-11e6-9a2e-f80931718f88.png)